src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/util/SSLUtil.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazon.opendistroforelasticsearch.security.ssl.util;
+
+import io.netty.buffer.ByteBuf;
+
+import java.nio.ByteOrder;
+
+
+public class SSLUtil {
```

Comment:
It looks like an utility class. All method should be static and it must have private constructor. `SSLUtil` is not a good name. Rename it to `TLS` and make `private static` under `OpenDistroPortUnificationHandler`.

Reply:
done